### PR TITLE
PEP 650: Correct minor grammar mistake

### PR DESCRIPTION
**Pull request by smwhite2**
Correct a few minor grammar mistakes.

### CODE REVIEW

**Comment by CAM-Gerlach** [comment]
As mentioned in our Readme, Contributing Guide, PEP 1, PEP 12, etc., non-Draft Standards Track PEPs are generally considered historical documents and not generally modified after acceptance/rejection/withdrawal, outside of rendering/syntax errors or other special circumstances. In particular, this is a Withdrawn PEP, so there is little reason to change it, and it is far more valuable to direct the efforts of contributors and reviewers toward PEPs that are Draft or Active, where the changes are much more likely to mean something.

IMO, the conforming of typographical quotes to apostrophes (single quotes) adds diff noise for no meaningful change in the rendered output. As such, from a PEP editor perspective I'm not so fond of it at least as-is, but as it does also fix a small typo, I will leave it to the PEP author and editor @brettcannon to make the the final call.

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
I honestly don't think the updates are really worth it, especially since the PEP was rejected. But since I got pulled into the PR to see what's going on I approved it. Although the lack of signed CLA makes it basically moot for now.

**Comment by CAM-Gerlach** [comment]
Seems to be a bit of a drive-by contribution, as the OP hasn't responded in over a month, and this PR appears to be their only GitHub contribution of any kind...

File: pep-0650.rst [hunk 1]
```diff
@@ -111,7 +111,7 @@ resources it takes to do so.
 
 Via this specification, we can enable a provider-supported
 *universal installer* to invoke the user-desired *installer backend*
-without the provider’s platform needing to have specific knowledge of
+without the provider's platform needing to have specific knowledge of
```
Comment (merwok) [inline pep-0650.rst:114]: No reason to change valid apostrophes.

**Comment by merwok** [comment]
Sorry, this is too small to be worth it.
If there were multiple instances of `There’s is` typo, a PR fixing all of them could be accepted.

Thanks for wanting to contribute regardless!  If you look through issues and find something you’d like to help with, feel free to tag us.

**Pull request closed without merging**